python/felis/datamodel.py

```diff
@@ -305,14 +327,14 @@ class SchemaVersion(BaseModel):
     current: str
     """The current version of the schema."""
 
-    compatible: list[str] | None = None
+    compatible: list[str] = Field(default_factory=list)
     """The compatible versions of the schema."""
 
-    read_compatible: list[str] | None = None
+    read_compatible: list[str] = Field(default_factory=list)
```

Comment:
How does this change interact with @andy-slac's concerns?

Reply:
This change just makes the variable return an empty list by default rather than a `None`, which seems more user friendly. It should not have any effect on the update for making version itself (a separate field) readable as a single string rather than forcing it to be a list.

Reply:
I think this is OK, we do not have any existing uses of that for now, so we can be free to use empty list instead of `None` as default.